workflow/airflow_libero/README.md

```diff
@@ -1,110 +0,0 @@
-# Airflow Example Usage
```

Comment:
README is removed but not added elsewhere, was it a pure duplicate?

Reply:
We need to update the main README generally. This was out of date though.